tests/test_bookmark.py

```diff
@@ -0,0 +1,76 @@
+import os
+import unittest
+from datetime import datetime as dt
+from datetime import timedelta
+
+from base import CloseioBase
+from tap_tester.base_suite_tests.bookmark_test import BookmarkTest
+
+class CloseioBookmarkTest(BookmarkTest, CloseioBase):
+    """Closeio bookmark test implementation"""
+
+
+    @staticmethod
+    def name():
+        return "tt_closeio_bookmark"
+
+    def streams_to_test(self):
+        expected_streams = self.expected_metadata().keys()
+        return set(expected_streams).difference(['event_log', 'activities'])
```

Comment:
I'm surprised this works.

I would've thought a set difference requires two sets

Reply:
changed this similar to all_fields_test to maintain similar approach through out.